src/ripple/app/ledger/impl/InboundLedgers.cpp

```diff
@@ -200,14 +201,19 @@ class InboundLedgersImp
         return ret;
     }
 
-    void logFailure (uint256 const& h)
+    void logFailure (uint256 const& h, std::uint32_t seq)
     {
-        mRecentFailures.insert (h);
+        ScopedLockType sl (mLock);
+
+        mRecentFailures.emplace(h, seq);
```

Comment:
It could be renamed. Generally, "logging" and "recording" are considered synonymous, but I agree that the term "log" tends to suggest that a record is made for human consumption rather than an automated tracking entry. This part of the code was only slightly touched by this change.